### DISCUSSION TITLE
GnuTLS: Error in the pull function

### DISCUSSION THREAD

**Discussion by davecrump** [community/community#70831] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

There seems to be a problem with downloading from GitHub in some repositories.  I get:

`wget https://github.com/BritishAmateurTelevisionClub/ryde-build/archive/master.zip
--2023-10-22 15:42:37--  https://github.com/BritishAmateurTelevisionClub/ryde-build/archive/master.zip
Resolving github.com (github.com)... 140.82.121.3
Connecting to github.com (github.com)|140.82.121.3|:443... connected.
GnuTLS: Error in the pull function.
Unable to establish SSL connection.
`

But in another repo, same system, it works:

` wget https://github.com/eclispe/ryde-utils/archive/master.zip
--2023-10-22 14:48:08--  https://github.com/eclispe/ryde-utils/archive/master.zip
Resolving github.com (github.com)... 140.82.121.4
Connecting to github.com (github.com)|140.82.121.4|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://codeload.github.com/eclispe/ryde-utils/zip/refs/heads/master [following]
--2023-10-22 14:48:08--  https://codeload.github.com/eclispe/ryde-utils/zip/refs/heads/master
Resolving codeload.github.com (codeload.github.com)... 140.82.121.9
Connecting to codeload.github.com (codeload.github.com)|140.82.121.9|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [application/zip]
Saving to: ‘master.zip’

master.zip                                  [ <=>                                                                         ]  22.77K  --.-KB/s    in 0.02s

2023-10-22 14:48:09 (1.20 MB/s) - ‘master.zip’ saved [23321]`

Any ideas?

Dave

**Comment by ebubekirgungor** [comment]
Hello, you can ignore ssl check with `wget --no-check-certificate`

**Reply by davecrump** [reply]
Thanks.  I think that GitHub have fixed the problem now, but good to know.

Dave
